dd-sdk-android/src/main/java/datadog/trace/common/writer/LoggingWriter.java

```diff
@@ -1,23 +1,12 @@
 package datadog.trace.common.writer;
 
-import static datadog.trace.common.serialization.JsonFormatWriter.TRACE_ADAPTER;
-
 import datadog.opentracing.DDSpan;
 import java.util.List;
 
 public class LoggingWriter implements Writer {
```

Comment:
I thin we could actually rename this to NoOpWriter ??

Reply:
In theory yes, but it's NoOp because every log is removed from files automatically. 
Because we might pull in new classes or lines using this class name, we want to keep it compatible. 
We'll rename it NoOpWriter when we rewrite this all ourselves.